src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -892,7 +892,17 @@ private void CreateGroup(object parameters)
 
         private bool CanCreateGroup(object parameters)
         {
-            return true;
+            return DynamoSelection.Instance.Selection.OfType<ModelBase>().Any();
+        }
+
+        private void UngroupNode(object parameters)
+        {
+            WorkspaceViewModel.DynamoViewModel.UngroupModelCommand.Execute(null);
+        }
+
+        private bool CanUngroupNode(object parameters)
+        {
+            return DynamoSelection.Instance.Selection.OfType<NodeModel>().Any();
```

Comment:
Shouldn't we be looking only specifically for this `NodeModel`, not just any `NodeModel`?  Maybe I'm mistaken...